src/components/about/About.tsx

```diff
@@ -74,6 +74,9 @@ const TermsOfService = () => (
         running, but we are volunteers, and we rely on third-parties, so we can't guarantee
         that <code>signalstickers.com</code> will be accessible at any time.
       </p>
+      <p>
+        We do not allow the inclusion of <code>signalstickers.com</code> in smartphone applications.
```

Comment:
/nit I would consider using broader language here, something like:

"The inclusion, in whole or in part, of the signalstickers.com website in any other application, website, or software is strictly forbidden."